app/routes.py

```diff
@@ -95,3 +95,41 @@ def get_id(kml_admin_id):
         ),
         200
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['DELETE'])
+def delete_id(kml_admin_id):
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    item = enforcer.get_item(kml_admin_id)
+
+    # Fetching a non existing Item will return "None"
+    if item is None:
+        logger.error("Could not find the following kml id in the database: %s", kml_admin_id)
+        abort(400, f"Could not find {kml_admin_id} within the database.")
+
+    file_id = item['file_id']
+
+    executor = S3FileHandling(AWS_S3_REGION_NAME, AWS_S3_ENDPOINT_URL)
+    executor.delete_file_in_bucket(AWS_S3_BUCKET_NAME, file_id)
+
+    enforcer.delete_item(kml_admin_id)
+
+    return make_response(
+        jsonify(
+            {
+                'success': True,
+                'id': kml_admin_id,
+                'links':
+                    {
+                        'self': f'{request.host_url}/kml/{item["admin_id"]}',
+                        'kml': f'{KML_STORAGE_URL}/{item["file_id"]}'
+                    }
+            }
+        ),
+        200
+    )
```

Comment:
I am not exactly sure, if this response really makes much sense. The returned links do no longer exist, and trying to follow them will lead to 404s. So probably we should simply return something like:
`"description": f'The kml {item["admin_id"]} was successfully deleted.'` instead of returning the links (self and kml). We should however leave the "success", "id" and the "200" in place.
Agree @IsabelleBzr @ltshb @pakb @boecklic?

Reply:
👍🏼  The answer should be:
```json
{
  "success": True,
  "id": kml_admin_id,
  "message": f'The kml {item["admin_id"]} was successfully deleted.'
}
```
I would use `message` instead of `description` to be consistent with the naming of error message (see https://github.com/geoadmin/doc-guidelines/blob/master/API.md#errors)